client.go

```diff
@@ -256,7 +196,11 @@ func (c *Client) Close() error {
 
 // Ping verifies that the underlying connection is healthy.
 func (c *Client) Ping() error {
-	row, err := c.node.Query("SELECT true")
+	timeout := 1 * time.Second
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	defer cancel()
```

Comment:
we should update the prototype of this method to allow a `ctx` in parameter, it feels weird to force the cancellation at `1 second`, we should let the user customize

Reply:
What about adding a new method `PingContext` that takes a `ctx`, leaving this one as a sane default ?